middle_end/flambda2/types/grammar/type_grammar.ml

```diff
@@ -487,7 +488,7 @@ and free_names_function_type (function_type : _ Or_unknown_or_bottom.t) =
   match function_type with
   | Bottom | Unknown -> Name_occurrences.empty
   | Ok { code_id; rec_info } ->
-    Name_occurrences.add_code_id (free_names rec_info) code_id Name_mode.normal
```

Comment:
Why this change ? Code IDs are basically symbols, they should never have mode In_types